### PR TITLE
docs: improved the documentation

### PR DESCRIPTION
**Pull request by jacobdotcosta**
Use the `pom.xml` as the source for the version number

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,11 +103,11 @@ If using resource filtering use build it this way instead:
 
 and then run with:
 
-    java -jar target/snowdrop-bot-0.1-SNAPSHOT-runner.jar
+    java -jar target/snowdrop-bot-$(xpath -q -e  "/project/version/text()" pom.xml)-runner.jar
```
Comment (iocanel) [inline README.md:106]: Curious, how does this render?

**Review by iocanel — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (iocanel) [reply README.md:106]: Mindblowing!!!

Comment (jacobdotcosta) [reply README.md:106]:
```bash
$ xpath -q -e  "/project/version/text()" pom.xml
0.9-SNAPSHOT
```

**Pull request merged**